app/src/main/java/com/iyanuoluwa/imovie/api2/CreditsJSon.kt

```diff
@@ -0,0 +1,10 @@
+package com.iyanuoluwa.imovie.api2
+
+
+import com.google.gson.annotations.SerializedName
+
+data class CreditsJSon(
```

Comment:
Rename this class to `Credits`
```suggestion
data class Credits(
```